examples/examples.test.ts

```diff
@@ -33,7 +40,7 @@ function validateAgainstSchemas(vlspec) {
   assert(isVgValid);
 }
 
-function getExampleList(examples) {
+export function getExampleList(examples) {
```

Comment:
Good point.  I was a function that you should have removed when we changed it to read directly from directory.


Done.